lib/atoms/div/example.html

```diff
@@ -0,0 +1 @@
+<div class="{{class}}"></div>
```

Comment:
I think we don't need this atom, because <div> will generally be a container of other components.